packages/moonstone/MoonstoneDecorator/fontGenerator.js

```diff
@@ -11,8 +11,8 @@ const fontName = 'Moonstone LG Display';
 // Locale Configuration Block
 const fonts = {
 	'NonLatin': {
-		regular: 'LG Display-Light',
-		bold:    'LG Display-Regular'
+		regular: 'LGSmartUIGlobal-Light',
+		bold:    'LGSmartUIGlobal-Regular'
```

Comment:
One question, Isn't it should be `LG Smart UI Global`?

Reply:
If it refers to just the family name, we can't target specific font-variants.